pipeline_dp/dp_computations.py

```diff
@@ -435,3 +436,30 @@ def compute_dp_var(count: int, normalized_sum: float,
         dp_mean += compute_middle(dp_params.min_value, dp_params.max_value)
 
     return dp_count, dp_mean * dp_count, dp_mean, dp_var
+
+
+def compute_dp_count_noise_params(
+        dp_params: MeanVarParams) -> Tuple[str, float]:
+    """Computes noise type and standard deviation for DP count.
+
+    Args:
+        dp_params: parameters for DP count aggregation.
+
+    Returns:
+        Tuple (noise type ('laplace' or 'gaussian'), noise std).
```

Comment:
nit: "stdv".

Also, I thought we were going with scale instead of stdv?

Reply:
Docstring is changed